runbot_gitlab_minimal/controllers/gitlab_ci_controller.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Vauxoo info@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import datetime
+import pprint
+import logging
+
+import openerp
+from openerp import http, SUPERUSER_ID
+from openerp.http import request
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotCIController(http.Controller):
+
+    @http.route(['/runbot/hook_gitlab/<int:repo_id>',
+                 '/runbot/hook_gitlab/org'], type='json', auth="public",
+                website=True)
+    def hook(self, repo_id=None, **post):
```

Comment:
@hugho-ad send me the following [code](https://github.com/Vauxoo/yoytec/blob/8.0/yoytec_checkout/controllers/main.py#L575-L581) to learn to use inherit over a controller

Could you verify if is posible use the original runbot controller [openerp.addons.runbot.runbot.hook](https://github.com/Vauxoo/odoo-extra/blob/397ba6d17e474f9b841860fcf0aae8a8243d8903/runbot/runbot.py#L1306) with a `request.jsonrequest` mapping keys from gitlab to github in order to use less code as short as possible?